lib/elastic_record/model/callbacks.rb

```diff
@@ -2,12 +2,12 @@ module ElasticRecord
   module Model
     module Callbacks
       def self.included(base)
-        return unless base.respond_to?(:after_save) &&  base.respond_to?(:after_destroy)
+        return unless base.respond_to?(:after_commit)
```

Comment:
elastic_record only supports Ruby 2.7+: https://github.com/data-axle/elastic_record/blob/3a2de444e2d6e3781f65f09611d30f5ba06d24a9/elastic_record.gemspec#L9

Based on https://www.fastruby.io/blog/ruby/rails/versions/compatibility-table.html, that pretty much limits it to Rail >= 6.

So if the point of this line is to ensure that we're using a version of Rails that supports `after_commit` (which has been around since Rails 3), we can remove it.

Otherwise, this is the same as `if defined?(ActiveRecord::Base) && base < ActiveRecord::Base`, which I think is a more explicit way of expressing what we're trying to achieve here.